mediaflow_proxy/extractors/base.py

```
@@ -1,48 +1,121 @@
+                        **kwargs,
+                    )
+
+                    if raise_on_status:
+                        try:
+                            response.raise_for_status()
+                        except httpx.HTTPStatusError as e:
+                            # Provide a short body preview for debugging
+                            body_preview = ""
+                            try:
+                                body_preview = e.response.text[:500]
+                            except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)